Tests/SwiftLintFrameworkTests/SwitchCaseAlignmentRuleTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SwitchCaseAlignmentRuleTests.swift
+//  SwiftLint
+//
+//  Created by Shai Mishali on 4/24/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+@testable import SwiftLintFramework
+import XCTest
+
+class SwitchCaseAlignmentRuleTests: XCTestCase {
+    func testSwitchCaseAlignment() {
```

Comment:
you can remove `testSwitchCaseAlignment` from `RulesTests` to avoid testing this twice

Reply:
What do you mean by this exactly? I sort of "Stole" this from a different rule (vertical spacing IMO)

Reply:
There's already a test that test exactly this on `RulesTests.swift`. You should remove it from there to avoid testing the same thing twice